src/storage/txn/scheduler.rs

```diff
@@ -475,6 +475,9 @@ fn gen_command_lock(latches: &Latches, cmd: &Command) -> Lock {
             let keys: Vec<&Key> = key_locks.iter().map(|x| &x.0).collect();
             latches.gen_lock(&keys)
         }
+        Command::ResolveLockLite {
+            ref resolve_keys, ..
+        } => latches.gen_lock(resolve_keys),
         Command::Commit { ref keys, .. } | Command::Rollback { ref keys, .. } => {
             latches.gen_lock(keys)
         }
```

Comment:
How about expand the wildcard `_` below so that compilation will fail when we adds new commands but forget to add it here?

Reply:
And please consider add the new command to `test_command_latches`

Reply:
done